lib/ReactViews/DragDropFile.tsx

```diff
@@ -119,7 +119,7 @@ class DragDropFile extends React.Component<PropsType> {
           [Styles.isActive]: this.props.viewState.isDraggingDroppingFile
         })}
       >
-        {isDefined(this.props.viewState.isDraggingDroppingFile) ? (
+        {this.props.viewState.isDraggingDroppingFile ? (
```

Comment:
This resulted in the drag-drop container being shown even when the user is not dragging anything.